src/styles/abstracts/_mixins.scss

```diff
@@ -44,3 +44,14 @@
   justify-content: center;
   align-items: center;
 }
+
+@mixin button-hover-animation($background-color) {
+  &:hover {
+    background: darken($background-color, 8%);
+    transition: all 0.3s ease;
```

Comment:
Do you think we need another `mixin` for transition? @LilBank @Kittinske15 
Ref: http://zerosixthree.se/8-sass-mixins-you-must-have-in-your-toolbox/
 
Example:
```
@mixin transition($what: all, $time: 0.2s, $how: ease-in-out) {
  -webkit-transition: $what $time $how;
  -moz-transition:    $what $time $how;
  -ms-transition:     $what $time $how;
  -o-transition:      $what $time $how;
  transition:         $what $time $how;
}
```

Usage:
 ```@include transition(all, 0.3s, ease);```

Reply:
Hm, @kimvcp idea looks better for color transitions. Thank you for pointing out. I am plus one to this comment.

Reply:
That's very interesting. For the name, `button-hover` sounds good, I'll change it then. Do you mean to replace `button-hover` with `@mixin transition`?. If so, however, I'd like the button to be darker when hovering as well. As in my function already included `transition: all 0.3s ease;` without adding extra `@mixin`.

Reply:
Nope, my point is to make another `@mixin` for `transition`, then replace your `transition: all 0.3s ease;` with the above usage (`@include transition(all, 0.3s, ease);`).

Reply:
Oh, that's great. It's a safer idea.